bats/fb-content-katello.bats

```diff
@@ -211,32 +215,44 @@ setup() {
 }
 
 @test "install katello-agent" {
+  tSkipIfPulp2
```

Comment:
The naming of this feels a bit odd. This reads like it should skip katello-agent tests if it's Pulp 2 but the implementation suggests it's skipping unless Pulp 2 is present